Makefile

```diff
@@ -204,7 +204,7 @@ $(OPENSHIFT_GOIMPORTS):
 
 .PHONY: imports
 imports: $(OPENSHIFT_GOIMPORTS) verify-go-versions
-	$(OPENSHIFT_GOIMPORTS) -m github.com/kcp-dev/kcp
+	$(OPENSHIFT_GOIMPORTS) -m github.com/kcp-dev/edge-mc
```

Comment:
This is an improvement, but still leaves the imports looking odd.  It now sorts imports from `kcp-dev/kcp` _ahead_ of imports from `k8s.io`.  I am OK with committing this and working further improvements in a separate PR.

Reply:
ok.  I noticed the same